client/src/components/SentimentAnalysis/Helpers.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const HEADLINE_PLACEHOLDER =
+  'i.e. Fighting Climate Change | Pete Buttigieg 2020 | Learn More';
+const CONTENT_PLACEHOLDER =
+  "i.e. Climate change is a life and death issue for our generation. It's time for bold action.";
+
+const INPUT_LIST = [
+  { label: 'HEADER', placeholder: HEADLINE_PLACEHOLDER },
+  { label: 'CONTENT', placeholder: CONTENT_PLACEHOLDER },
+];
+
+const AnalysisInput = props => {
```

Comment:
For the future: 
- I'm not sure if this is searching through ads in our db and just using the lang analysis results already computed or calling the api and recomputing (haven't read all the way through yet.)
  - If you are doing the latter, consider looking into adding captcha + a 1000 character limit (that's how big one unit of an api call is for the google api.)

Reply:
Really good point, I'm doing the latter one and this comment is really useful to me, thank you! :)